cmd/create/idp/cmd.go

```diff
@@ -434,6 +438,24 @@ func getIDPName(cmd *cobra.Command, idpName string, r *rosa.Runtime) string {
 	return strings.Trim(idpName, " \t")
 }
 
+func ValidateIdpName(idpName interface{}) error {
```

Comment:
Do we have similar validations in the API?

Reply:
i dont think so

Reply:
awesome, thanks for the context. Okay makes sense to just have it here for this use case.